identifiers/country-in/lok_sabha.csv

```diff
@@ -1,544 +1,544 @@
+ocd-division/country:in/state:gj/cd:junagadh,Gujarat Lok Sabha constituency Junagadh
+ocd-division/country:in/state:gj/cd:amreli,Gujarat Lok Sabha constituency Amreli
+ocd-division/country:in/state:gj/cd:bhavnagar,Gujarat Lok Sabha constituency Bhavnagar
+ocd-division/country:in/state:gj/cd:anand,Gujarat Lok Sabha constituency Anand
+ocd-division/country:in/state:gj/cd:kheda,Gujarat Lok Sabha constituency Kheda
+ocd-division/country:in/state:gj/cd:panchmahal,Gujarat Lok Sabha constituency Panchmahal
+ocd-division/country:in/state:gj/cd:dahod,Gujarat Lok Sabha constituency Dahod (ST)
+ocd-division/country:in/state:gj/cd:vadodara,Gujarat Lok Sabha constituency Vadodara
+ocd-division/country:in/state:gj/cd:chhota_udaipur,Gujarat Lok Sabha constituency Chhota Udaipur (ST)
+ocd-division/country:in/state:gj/cd:bharuch,Gujarat Lok Sabha constituency Bharuch
+ocd-division/country:in/state:gj/cd:bardoli,Gujarat Lok Sabha constituency Bardoli (ST)
+ocd-division/country:in/state:gj/cd:surat,Gujarat Lok Sabha constituency Surat
+ocd-division/country:in/state:gj/cd:navsari,Gujarat Lok Sabha constituency Navsari
+ocd-division/country:in/state:gj/cd:valsad,Gujarat Lok Sabha constituency Valsad (ST)
+ocd-division/country:in/state:hr/cd:ambala,Haryana Lok Sabha constituency Ambala (SC)
+ocd-division/country:in/state:hr/cd:kurukshetra,Haryana Lok Sabha constituency Kurukshetra
+ocd-division/country:in/state:hr/cd:sirsa,Haryana Lok Sabha constituency Sirsa (SC)
+ocd-division/country:in/state:hr/cd:hissar,Haryana Lok Sabha constituency Hissar
+ocd-division/country:in/state:hr/cd:karnal,Haryana Lok Sabha constituency Karnal
+ocd-division/country:in/state:hr/cd:sonipat,Haryana Lok Sabha constituency Sonipat
+ocd-division/country:in/state:hr/cd:rohtak,Haryana Lok Sabha constituency Rohtak
+ocd-division/country:in/state:hr/cd:bhiwani-mahendragarh,Haryana Lok Sabha constituency Bhiwani-Mahendragarh
+ocd-division/country:in/state:hr/cd:gurgaon,Haryana Lok Sabha constituency Gurgaon
+ocd-division/country:in/state:hr/cd:faridabad,Haryana Lok Sabha constituency Faridabad
+ocd-division/country:in/state:hp/cd:kangra,Himachal Pradesh Lok Sabha constituency Kangra
+ocd-division/country:in/state:hp/cd:mandi,Himachal Pradesh Lok Sabha constituency Mandi
+ocd-division/country:in/state:hp/cd:hamirpur,Himachal Pradesh Lok Sabha constituency Hamirpur
+ocd-division/country:in/state:hp/cd:shimla,Himachal Pradesh Lok Sabha constituency Shimla (SC)
+ocd-division/country:in/state:jk/cd:baramulla,Jammu and Kashmir Lok Sabha constituency Baramulla
+ocd-division/country:in/state:jk/cd:srinagar,Jammu and Kashmir Lok Sabha constituency Srinagar
+ocd-division/country:in/state:jk/cd:anantnag,Jammu and Kashmir Lok Sabha constituency Anantnag
+ocd-division/country:in/state:jk/cd:ladakh,Jammu and Kashmir Lok Sabha constituency Ladakh
+ocd-division/country:in/state:jk/cd:udhampur,Jammu and Kashmir Lok Sabha constituency Udhampur
+ocd-division/country:in/state:jk/cd:jammu,J
```

Comment:
We need to think of a way to handle the two anglo-indian reserved seats. These are nominated for anglo-indian community that has no native state of its own. https://en.wikipedia.org/wiki/Anglo-Indian_reserved_seats_in_the_Lok_Sabha

Reply:
Added 2 OCDIDs for the anglo-indian reserved seats.